packages/medusa/src/services/__tests__/gift-card.js

```diff
@@ -0,0 +1,253 @@
+import { IdMap, MockManager, MockRepository } from "medusa-test-utils"
+import mockRepository from "medusa-test-utils/dist/mock-repository"
```

Comment:
Remove import

Reply:
We can default to 20, but shouldn't we be consistent across the solution? (other places we used 50, so maybe also change this?)